openedx/features/calendar_sync/tests/test_ics.py

```diff
@@ -65,9 +65,9 @@ def expected_ics(self, *assignments):
 METHOD:REQUEST
 BEGIN:VEVENT
 SUMMARY:{summary}
-DTSTART;VALUE=DATE-TIME:{timedue}
+DTSTART:{timedue}
```

Comment:
The date format has been shortened in icalendar package.
See this PR for reference: https://github.com/collective/icalendar/pull/479/files